### ISSUE TITLE
About the hyperparameter for CT image registration.

### ISSUE THREAD

**Issue by MingR-Ma** [junyuchen245/TransMorph_Transformer_for_Medical_Image_Registration#42]
Hi, I would like to ask what the hyperparameter of regularization used when using VoxelMorph as well as TransMorph for CT image registration is. What is the similarity loss function? Looking forward to your answer.
